csharp/ql/test/library-tests/dataflow/fields/J.cs

```diff
@@ -27,6 +29,14 @@ private void M1()
         Sink(r4.Prop2); // no flow
     }
 
+    private void M2()
+    {
+        var o = Source<object>(1);
```

Comment:
Change `1` to `2` (and in the inline expectation comment below).

Reply:
Good idea! I will do that!